cfme/tests/infrastructure/test_timelines.py

```diff
@@ -121,24 +129,27 @@ def test_vm_event(provider, gen_events, test_vm):
     def nav_step():
         test_vm.load_details()
         toolbar.select('Monitoring', 'Timelines')
-    wait_for(count_events, [test_vm.name, nav_step], timeout=60, fail_condition=0,
+
+    wait_for(count_events, [test_vm, nav_step], timeout='3m', delay=10, fail_condition=0,
              message="events to appear")
 
 
 @pytest.mark.ignore_stream("upstream")
 @pytest.mark.meta(blockers=[1281746])
-def test_cluster_event(provider, gen_events, test_vm):
+def test_cluster_event(provider, gen_events, test_vm, rest_api):
     """Tests cluster event on timelines
 
     Metadata:
         test_flag: timelines, provision
     """
     def nav_step():
-        test_vm.load_details()
-        pytest.sel.click(InfoBlock.element('Relationships', 'Cluster'))
+        cluster_id = int(rest_api.collections.vms.get(name=test_vm.name).ems_cluster_id)
```

Comment:
Can we modify this to call it vm_cluster_id, and I think that we should move this into the vm object again, other people may want to get the cluster_id, so moving it there is a good option......then the line below becomes,
```cluster_name = [cl.name for cl in get_all_clusters() if cl.id == test_vm.get_cluster_id()][-1]```
Also, I'm assuming (bad idea I know) that get_all_clusters() returns Cluster objects, in which case, do we need to create a new Cluster object on line 149, couldn't we just use ```cl``` as the object?